hw/ip/otbn/dv/otbnsim/sim/sim.py

```diff
@@ -117,6 +117,8 @@ def step(self, verbose: bool) -> Tuple[Optional[OTBNInsn], List[Trace]]:
         returns no instruction and no changes.
 
         '''
+        if self.state.wsrs.URND.start_prng:
+            self.state.wsrs.URND.step_URND()
```

Comment:
Probably nicer to call the step function unconditionally (but probably just when we're running?) and then put the "should I step URND" logic into the WSR code.